src/database/config/config.json

```diff
@@ -3,7 +3,7 @@
     "username": "root",
     "password": null,
     "database": "database_development",
-    "host": "127.0.0.1",
+    "host": "localhost",
     "dialect": "mysql"
```

Comment:
why do you have mysql instead of postgres?

Reply:
> why do you have mysql instead of postgres?

config.json deleted because it's outdated